src/pybind/mgr/orchestrator.py

```diff
@@ -980,6 +980,10 @@ def update_rgw(self, spec):
         """
         raise NotImplementedError()
 
+    def upgrade_check(self, image_name):
+        # type: (string) -> Completion
```

Comment:
#32010 will show that this signature is invalid.
```suggestion
        # type: (str) -> Completion
```
(This line was the reason I made #32010 )